src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+
+        // initialise an array of tasks that is <=100
+        //sizeOfList is the number of items in the list
+        Task[] listOfTasks = new Task[100];
+        int sizeOfList = 0;
+
+        // Reads in the first command from the user
+        Scanner scanner = new Scanner(System.in);
+        String inputCommand = scanner.nextLine();
+
+        while (inputCommand.equals("bye") == false) {
+
+            if (inputCommand.equals("list")) {
+                // prints out the list
+                printsOutTheList(listOfTasks, sizeOfList);
+
+            } else if (inputCommand.contains("done ") == true) {
+                // marks task in the stated index as done
+                indicateTaskAsDone(listOfTasks, inputCommand);
+
+            } else {
+                // adds a new task into the list
+                sizeOfList = addInNewTask(listOfTasks, sizeOfList, inputCommand);
+
+            }
+            // take in the next command
+            inputCommand = scanner.nextLine();
+        }
+
+        printExitMessage();
+    }
+
+    /**
+     * Prints the exit message
+     */
+    private static void printExitMessage() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
```

Comment:
Since you print the same line several times, how about making it into a constant and calling it?

Reply:
I do agree with this suggestion.